specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -2431,76 +2160,81 @@
           "items": {
             "type": "string"
           },
-          "description": "Availability zones for nodes. Must use VirtualMachineScaleSets AgentPoolType."
+          "description": "The list of Availability zones to use for nodes. This can only be specified if the AgentPoolType property is 'VirtualMachineScaleSets'."
         },
         "enableNodePublicIP": {
           "type": "boolean",
           "description": "Enable public IP for nodes"
```

Comment:
We need more details here. When do I want to use this? Why? Is there an existing article we can link to?

Reply:
https://docs.microsoft.com/en-us/azure/availability-zones/az-overview#availability-zones
 
 Not clear to me if we ever default you to using all availiability zones if not specified.

Reply:
I was actually asking about `enableNodePublicIP` - but I found there's a document here about it: https://docs.microsoft.com/en-us/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools